components/src/Tables/Table.js

```diff
@@ -1,38 +1,98 @@
 import React from 'react';
 import styled from 'styled-components';
+import { colour, space, font, radius, shadow } from '@nulogy/tokens';
+import QuietButton from '../Button/QuietButton';
 
-const borderStyle = ({ theme }) => `1px solid ${theme.colour.neutral[400]}`;
+const borderWidth = 1;
+
+const withoutBorder = value => value - borderWidth;
+
+const lineHeight = name => font.lineHeight[name] * font.size.medium;
 
 export const Cell = styled.td`
-  padding: 1rem;
-  text-align: left;
-  border-bottom: ${borderStyle};
+  font-size: ${font.size.small}px;
+  line-height: ${lineHeight('smaller') / font.size.small};
+  padding: ${space.x2}px ${space.x1}px;
+  &:first-child{
+    padding-left: ${space.x2}px;
+  }
+  &:last-child{
+    padding-right: ${space.x2}px;
+    width:72px; // not the right place
+  }
 `;
 
-export const Row = styled.tr``;
+export const ActionCell = styled(Cell)`
+  padding: ${space.x1}px ${space.x2}px ${space.x1}px ${space.x1}px;
```

Comment:
a few comments here:

1. the right padding of `space.x2` is already set on the last-child td above and shouldn't be necessary.
2. The only things being overridden here are the top and bottom padding. I would prefer to see only those values set.